src/store/index.ts

```diff
@@ -1,57 +1,118 @@
-import { createStore, combineReducers, applyMiddleware } from 'redux';
+import { createStore, AnyAction } from 'redux';
 import { composeWithDevTools } from 'redux-devtools-extension';
-import thunk from 'redux-thunk';
-import { Dispatch } from 'react';
-
-import loadingReducer, { finishLoading, startLoading } from './loading';
-import messageReducer, { setMessage } from './message';
-import { fetchMessage } from '../helpers/api';
-
-/**
- * Each concrete reducer will receive all the actions but only its part of the state
- *
- * const rootReducer = (state = {}, action) => ({
- *   loading: loadingReducer(state.loading, action),
- *   message: messageReducer(state.message, action),
- * })
- */
-const rootReducer = combineReducers({
-  loading: loadingReducer,
-  message: messageReducer,
-});
 
-// We automatically get types returned by concrete reducers
-export type RootState = ReturnType<typeof rootReducer>;
+// Action types - is just a constant. MUST have a unique value.
+const START_LOADING = 'START_LOADING';
+const INIT_POSTS = 'INIT_TODOS';
+const HANDLE_ERROR = 'HANDLE_ERROR';
+const DELETE_POST = 'DELETE_POST';
+const SET_QUERY = 'SET_QUERY';
+const SET_FILTER_QUERY = 'SET_FILTER_QUERY';
+
+export const startLoading = () => ({ type: START_LOADING });
+export const initPosts = (posts: {
+  postUser: UserFromServer
+  | undefined; postComment: CommentFromServer[];
+  id: number; title: string; body: string; userId: number;
+}[]) => ({
```

Comment:
```suggestion
export const initPosts = (posts: {
  postUser: UserFromServer | undefined; 
  postComment: CommentFromServer[];
  id: number; 
  title: string; 
  body: string; 
  userId: number;
}[]) => ({
```
each one on the new line